hooks/getFeatures.ts

```diff
@@ -0,0 +1,29 @@
+import { Manifest } from "@hyperion-framework/types";
+
+export const getBounds = (items) => {
+  return items.map(
+    (item) => {
+      const pointFeatures = item.navPlace.features.filter(
+        (feature) => feature.geometry.type === "Point")
+      const manifestCoordinates = pointFeatures.map((feature) => feature.geometry.coordinates.reverse())
+      // This isn't exactly right.  Fix.
+      return [...manifestCoordinates].shift()
+    }
+  )
+}
+
+export const getFeatures = (manifests: [Manifest]) => {
+  return manifests.map(
+    (manifest) => {
+      // Spread this ...
+      return ({
+        id: manifest.id,
+        thumbnail: manifest.thumbnail,
+        label: manifest.label,
+        slug: `works/${manifest.slug}`,
+        features: manifest.navPlace.features.filter(
+          (feature) => feature.geometry.type === "Point"),
+      })
+    }
+  );
+};
```

Comment:
Some updates here too. I'm not sure exactly how most projects organize this stuff but my understanding is that these technically are not hooks as they don't lean on React. They might fit better in `services/*` or `services/iiif/*` in some way.

Reply:
K.  Got it.  Sorry, still figuring out patterns.